app/containers/GameBoard.jsx

```diff
@@ -37,12 +38,7 @@ class GameBoard extends Component {
   }
 
   createGame() {
```

Comment:
since dealDeck was only used once, by createGame, and createGame didn't have much in it, it should be one method.